arches/app/models/sql/functions/README.md

```diff
@@ -0,0 +1,54 @@
+Adding an initial migration for existing database functions
+===========================================================
+
+- Recover the old function from the database.
+- Paste it into a new file under arches/app/models/migrations/sql/functions.
+- Add a SQLItem() in sql_config.py like::
+```python
+    SQLItem(
+        "__arches_instance_view_update",
+        format_file_into_sql("__arches_instance_view_update.sql", "sql/functions"),
+        reverse_sql="drop function __arches_instance_view_update;",
+        replace=True,
+    ),
+```
+- manage.py makemigrations --name <description>
+- Edit the migration number to follow the Arches convention of using issue numbers.
+- Edit the migration file to treat the CreateSQL operation as a state-only operation,
+  since the function already exists in the database.::
+```python
+    operations = [
+        migrations.operations.special.SeparateDatabaseAndState(
+            state_operations=[
+                django_migrate_sql.operations.CreateSQL(
+                    name="__arches_instance_view_update",
+                    sql="\nCREATE OR REPLACE FUNCTION ...
+                    reverse_sql="drop function __arches_instance_view_update;",
+                ),
+            ],
+        ),
+    ]
+```
+- Create a commit.
+- Make your changes to the function SQL file.
+- manage.py makemigrations
+- Move the operation from the second migration into the end of previous migration (as
+a top-level operation) and delete the second migration::
+```python
+    operations = [
+        migrations.operations.special.SeparateDatabaseAndState(
+            state_operations=[
+                django_migrate_sql.operations.CreateSQL(
+                    name="__arches_instance_view_update",
+                    sql="\nCREATE OR REPLACE FUNCTION ...
+                    reverse_sql="drop function __arches_instance_view_update;",
+                ),
+            ],
+        ),
+        django_migrate_sql.operations.AlterSQL(
+            name="__arches_instance_view_update",
+            sql="\ncreate or replace function __arches_instance_view_update() ...,
+            reverse_sql="\nCREATE OR REPLACE FUNCTION public.__arches_instance_view_update()...,
+            state_reverse_sql="drop function __arches_instance_view_update;",
```

Comment:
a summary of convo with @aarongundel:

`reverse_sql` is what will get executed on the db in reverse.

`state_reverse_sql` is for Django internals only, and here it's just echoing back what we passed to the `reverse_sql` arg of `SQLItem()`, which is just a dummy since we didn't / aren't maintaining a whole separate .sql file for the first reverse sql in the chain.